src/compiler/types.ts

```diff
@@ -1725,6 +1725,7 @@ namespace ts {
         getSymbolAtLocation(node: Node): Symbol;
         getSymbolsOfParameterPropertyDeclaration(parameter: ParameterDeclaration, parameterName: string): Symbol[];
         getShorthandAssignmentValueSymbol(location: Node): Symbol;
+        getExportSpecifierLocalTargetSymbol(location: ExportSpecifier): Symbol;

```

Comment:
it is not the symbol of the export, it is the think it aliases. others do not have this behavior.